nncf/config/extractors.py

```diff
@@ -11,64 +11,49 @@
  limitations under the License.
 """
 
-from copy import deepcopy
-from typing import Dict, List, Optional
+from typing import Dict
+from typing import List
+from typing import Optional
```

Comment:
We should discuss and choose one way to import from typing

Reply:
The way has been recommended in the style guide, but it is not enforceable automatically using IDE. Should make use of  `black` mandatory, if it is possible to adjust it so that it automatically groups typing imports specifically.

Reply:
Looks like `black` is too opinionated for its own good and its style cannot be adjusted at all.